### ISSUE TITLE
Requests

### ISSUE THREAD

**Issue by alvarofpp** [odufrn/odufrn-downloader#80]
# Requests
Criar uma classe que manipule as requisições realizadas durante o uso do pacote.

- **Tipo**: Refatoração.
- **Depende da issue**: #77.
  - **Status**: Totalmente.
- **Branch**: no final, deverá ser feito um merge para a branch `version-2`.

**Checklist**:
- [ ] Implementar feature;
- [ ] Adicionar testes;
- [ ] Atualizar _docstring_;
- [ ] Atualizar documentação.

## Para entender melhor
Atualmente existem dois métodos na classe **Core** que realizam requisições: `_load_list` e `_request_get`. É interessante colocar esse comportamento em uma classe própria, valendo até ser um mixin.

**Comment by diegodiogenes** [comment]
Concordo em ter uma classe própria, sobre o mixin não tenho certeza. :+1:
